cmd/ingress-operator/start.go

```diff
@@ -65,76 +73,66 @@ func NewStartCommand() *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringVarP(&options.ShutdownFile, "shutdown-file", "s", defaultTrustedCABundle, "file which when modified should trigger a shutdown. If empty, shutdown triggering is disabled.")
+	cmd.Flags().StringVarP(&options.ShutdownFile, "shutdown-file", "s", defaultTrustedCABundle, "if provided, shut down the operator when this file changes")
+	cmd.Flags().StringVarP(&options.MetricsListenAddr, "metrics-listen-addr", "", ":6000", "metrics endpoint listen address")
+	cmd.Flags().StringVarP(&options.OperatorNamespace, "namespace", "n", manifests.DefaultOperatorNamespace, "namespace the operator is deployed to")
+	cmd.Flags().StringVarP(&options.IngressControllerImage, "image", "i", "", "image of the ingress controller the operator will manage (required)")
+	cmd.Flags().StringVarP(&options.ReleaseVersion, "release-version", "", statuscontroller.UnknownVersionValue, "the release version the operator should converge to")
```

Comment:
An alternative approach, which we use in the router, is to read the environment variable to get a default value, and use a hard-coded value as the default if the environment variable is not set.

Reply:
Isn't this already what's effectively happening, since the env var is used for the CLI options in the deployment YAML?

I also went ahead and made this and most of the other options required with defaults.

Reply:
> Isn't this already what's effectively happening, since the env var is used for the CLI options in the deployment YAML?

Sort of, but if you used `os.Getenv` in the code, then the deployment wouldn't need to translate environment variables into command-line parameters, which could be convenient for people writing their own deployment definitions or running the operator directly (if such people exist).  Anyway, it isn't important; just a suggestion.

Reply:
Driving this via env variables would seem natural to me too. But, not important for now.